### PR TITLE
chore: fix version tests

### PR DESCRIPTION
**Pull request by gdbranco**

### CODE REVIEW

**Comment by gdbranco** [comment]
@andreadecorte We actually added it just a while back actually https://github.com/openshift/release/pull/35679

But we hit a few problems, the tests were not picking up PR changes, so we couldn't fix it. So we reverted and will plan out enabling back in a better fashion.

**Review by renan-campos — APPROVED** [review]
/lgtm

File: cmd/create/cluster/cmd_test.go [hunk 1]
```diff
@@ -26,34 +27,34 @@ var _ = Describe("Validates OCP version", func() {
...
-			v, err := client.ValidateVersion("4.12.0-0.nightly-2022-11-25-185455-nightly",
-				[]string{"4.12.0-0.nightly-2022-11-25-185455-nightly"}, nightly, false, true)
+			v, err := client.ValidateVersion("4.12.0-0.nightly-2023-04-10-222146",
+				[]string{"4.12.0-0.nightly-2023-04-10-222146"}, nightly, false, true)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(v).To(Equal("openshift-v4.12.0-0.nightly-2022-11-25-185455-nightly-nightly"))
+			Expect(v).To(Equal("openshift-v4.12.0-0.nightly-2023-04-10-222146-nightly"))
 		})
 
 		It("KO: Fails with a nightly version of OCP for hosted clusters "+
```
Comment (renan-campos) [inline cmd/create/cluster/cmd_test.go:43]:
```suggestion
		It("OK: Fails with a nightly version of OCP for hosted clusters "+
```

Comment (renan-campos) [reply cmd/create/cluster/cmd_test.go:43]: :boxing_glove:

Comment (renan-campos) [reply cmd/create/cluster/cmd_test.go:43]: Wait are the "KO"s intentional because these tests are simulating a failure?

Comment (gdbranco) [reply cmd/create/cluster/cmd_test.go:43]: @renan-campos Exactly, not sure who set it up haha

**Pull request merged**